package.json

```diff
@@ -263,6 +279,11 @@
                     "submenu": "aks.K8sAPIEndpointHealthCheck",
                     "when": "view == kubernetes.cloudExplorer && viewItem =~ /aks\\.cluster/i",
                     "group": "9@3"
+                },
+                {
+                    "submenu": "aks.inspektorGadgetSubMenu",
+                    "when": "view == kubernetes.cloudExplorer && viewItem =~ /aks\\.cluster/i",
```

Comment:
To enable it in both the cloud explorer and cluster explorer, I think you can use this (see ASO command):
```json
"when": "view == kubernetes.cloudExplorer && viewItem =~ /aks\\.cluster/i || view == extension.vsKubernetesExplorer && viewItem =~ /vsKubernetes\\.\\w*cluster$/i"
```